pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -60,7 +60,6 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
 	is := makeInjectionSet(binding, cred)
-
 	if err := b.injectSecret(is.secret); err != nil {
```

Comment:
this may just be a stylist thing, I'm not sure, but the level of indirection here confuses me.
We're calling "injectSecret()" with a secret.
- First, its not really "injecting" it into anything, its creating it in etcd, right? So it seems like it should be called createSecret or storeSecret - inject made me think it was inserting into some other object.
- Second, why do we have injectSecret() as a new func instead of just inlining those 2 lines of code? Seems like an unnecessary split of logic.

And while I'm at it :-)  why do we have an injectionSet at all? Why isn't it just a Secret?  There's just the Secret in there anyway.  Do we really need something between the Binding and the Secret?